mappings/net/minecraft/network/packet/c2s/play/BookUpdateC2SPacket.mapping

```diff
@@ -1,7 +1,7 @@
 CLASS net/minecraft/class_2820 net/minecraft/network/packet/c2s/play/BookUpdateC2SPacket
 	FIELD field_26897 slot I
-	FIELD field_34041 bookPages Ljava/util/List;
-	FIELD field_34042 bookTitle Ljava/util/Optional;
+	FIELD field_34041 pages Ljava/util/List;
+	FIELD field_34042 title Ljava/util/Optional;
 	METHOD <init> (ILjava/util/List;Ljava/util/Optional;)V
 		ARG 1 slot
 		ARG 2 bookPages
```

Comment:
Need to update the parameters and getter names as well.

Reply:
Just realized I didn't update the getters...

Reply:
Done